src/schema/v2/me/myCollectionInfo.ts

```diff
@@ -33,6 +38,35 @@ export const myCollectionInfoFields = {
     type: new GraphQLNonNull(GraphQLInt),
     resolve: ({ artists_count }) => artists_count,
   },
+  collectedArtistsConnection: {
+    description: "A connection for the artists in the users' collection",
+    type: artistConnection.connectionType,
+    args: pageable({}),
+    resolve: (_root, args, context) => {
+      const { collectionArtistsLoader } = context
+
+      if (!collectionArtistsLoader) return
+
+      const gravityArgs = omit(convertConnectionArgsToGravityArgs(args), [
+        "page",
+      ])
+      gravityArgs.total_count = true
+
+      return collectionArtistsLoader("my-collection", gravityArgs).then(
+        ({ body: collectedArtists, headers }) => {
+          const totalCount = parseInt(headers["x-total-count"] || "0", 10)
+
+          return {
+            totalCount,
+            ...connectionFromArraySlice(collectedArtists, args, {
```

Comment:
Let's use `paginationResolver` instead. Pagination in **Force** doesn't work correctly with `connectionFromArraySlice` ([example 1](https://github.com/artsy/metaphysics/pull/4091), [example 2](https://github.com/artsy/metaphysics/pull/3556), [pagination docs](https://github.com/artsy/metaphysics/blob/bbee9c90f14bad1e6c233980c53c06973569e1f5/docs/pagination.md#L1)).